lib/template.html.eex

```diff
@@ -1,9 +1,17 @@
-<script src="https://www.google.com/recaptcha/api.js?hl=<%= @options[:hl] %>" ></script>
+<script src="https://www.google.com/recaptcha/api.js?onload=onLoad&hl=<%= @options[:hl] %>" async defer></script>
```

Comment:
I'd rather specify the onload myself, like `<%= @options[:onload] %>`
so that all kinds of behavior can be linked